conbench/templates/compare-list.html

```diff
@@ -55,59 +55,44 @@ <h5 class="card-title"></h5>
 </div>
 {% endif %}
 <p>
-  The <code>change</code> column below shows the direct comparison between matching benchmark results.
+  <a data-bs-toggle="collapse"
+     href="#collapseHelpblock"
+     role="button"
+     aria-expanded="false"
+     aria-controls="collapseHelpblock">toggle help for using the table below</a>
 </p>
-<p>
-  The <code>z-score</code> column shows how a contender benchmark result performs in view of a historical ensemble of comparable
-  benchmark results (in this case, the reference run contains only the newest of the results in that ensemble;
-  all other results were submitted via different CI runs in the past).
-</p>
-<p>
-  When <code>change</code> and <code>z-score</code> have a negative sign then this is defined as degradation;
-  a positive sign corresponds to an improvement (this directionality is automatically derived from the unit string and may be wrong).
-</p>
-<p>
-  The <code>change</code> and <code>z-score</code> columns may provide contradicting opinions about improvement vs. degradation.
-  The lookback z-score method (documented
-  <a href="https://github.com/conbench/conbench/blob/52cf34f96cb471f493dc201b0a57ece6132eb10d/docs/pages/lookback_zscore.md">here</a>)
-  is supposed to provide the more credible result.
-</p>
-{% if plot_history %}
-  <div class="col-md-12">
-    <h6>Top Outliers</h6>
-    <div id="bokeh-carousel"
-         style="visibility: hidden"
-         class="carousel slide"
-         data-ride="carousel">
-      <div class="carousel-inner" role="listbox">
-        <ol class="carousel-indicators">
-          {% if plot_history|length > 1 %}
-            {% for plot in plot_history %}
-              {% if loop.index == 1 %}
-                <li data-target="#bokeh-carousel" data-slide-to="0" class="active"></li>
-              {% else %}
-                <li data-target="#bokeh-carousel" data-slide-to="{{ loop.index - 1 }}"></li>
-              {% endif %}
-            {% endfor %}
-          {% endif %}
-        </ol>
-        {% for plot in plot_history %}
-          <div class="item  {% if loop.index == 1 %}active{% endif %} ">
-            <div align="center">
-              <a href="{{ outlier_urls[loop.index - 1] }}">{{ outlier_names[loop.index - 1] }}</a>
-            </div>
-            <div id="plot-history-{{ loop.index - 1 }}" align="center"></div>
-          </div>
-        {% endfor %}
-        {% if plot_history|length > 1 %}
-          <br>
-          <br>
-          <br>
-        {% endif %}
-      </div>
-    </div>
+<div class="collapse mb-3" id="collapseHelpblock">
+  <div class="card card-body">
+    <p>
+      Each row in the table below represents a comparison. The outcome of that comparison is summarized by two metrics: <code>change</code> and <code>z-score</code>.
```

Comment:
```suggestion
      Each row in the table below represents a comparison. The outcome of that comparison is summarized by two metrics: <code>percent change</code> and <code>z-score</code>.
```
just to clarify this isn't absolute change (in seconds or whatever units)

Reply:
Er, this implies changing the name of the column. In that case it's clear since we put a `%` on the number. So maybe `percent <code>change</code>`? It's clarified below, but ideally it should be clear from the start

Reply:
Right, here I used the column headings, and I think that is what we want to do. That is, we can of course also change the column headings. I thought about `relchange` instead of `change`, but thought it's best do not do everything at once.